guides/testing-arquillian-junit5.adoc

```diff
@@ -0,0 +1,292 @@
+----
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-maven-plugin</artifactId>
+            <version>${version.wildfly-maven-plugin}</version>
+            <configuration>
+                <jboss-home>${jboss.home}</jboss-home>
+                <provisioning-dir>${jboss.home}</provisioning-dir>
+                <feature-packs>
+                    <feature-pack>
+                        <groupId>org.wildfly</groupId>
+                        <artifactId>wildfly-ee-galleon-pack</artifactId>
+                    </feature-pack>
+                </feature-packs>
+                <channels>
+                    <channel>
+                        <manifest>
+                            <groupId>org.wildfly.channels</groupId>
+                            <artifactId>wildfly-ee</artifactId>
+                        </manifest>
+                    </channel>
+                </channels>
+                <layers>
+                    <layer>ee-core-profile-server</layer>
+                    <layer>jpa</layer>
+                    <layer>h2-default-datasource</layer>
+                    <layer>transactions</layer>
+                </layers>
+                <galleon-options>
+                    <jboss-fork-embedded>true</jboss-fork-embedded>
+                </galleon-options>
+            </configuration>
+            <executions>
+                <execution>
+                    <id>provision-server</id>
+                    <goals>
+                        <goal>provision</goal>
+                    </goals>
+                    <phase>process-test-resources</phase>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+</build>
+----
+
+The above configuration will provision a server based with Jakarta EE Core Profile specifications, Jakarta Persistence,
+Jakarta Transactions and the default H2 data source. The layers can be removed to provision a full WildFly server.
+
+The provisioning is bound to the `process-test-resources` phase. This is the last phase before the `test` phase which
+is when our tests will be executed by default. We need a server before we can use Arquillian for our tests.
+
+== Writing Tests
+
+Now that our POM is configured, we can write a test for our application. The first step is to tell JUnit 5 we want
+to extend the functionality with Arquillian. The simplest approach is to annotate your test with `@ArquillainTest`. The
+other option is to annotate the test with `@ExtendWith(ArquillianExtension.class)`.
+
+[source,java]
+----
+@WildFlyArquillian
+public class AddTaskResourceTest {
+}
+----
+
+=== Client Test
+
+Arquillian can run both in the container or as a client. For the first example we will run as a client. When running as
+a client the test runs outside the container. The simplest way to run as a client is to use the `@RunAsClient`
+annotation.
+
+[source,java]
+----
+@WildFlyArquillian
```

Comment:
The same here, apparently, this should be `@ArquillianTest` instead.

Reply:
Good catch on these @yersan. Thank you!